scripts/gen_db.py

```
@@ -240,7 +240,33 @@ def insert_hadiths(conn: sqlite3.Connection, hadiths_df: pl.DataFrame) -> None:
+
+    # Return the translation if found, otherwise None
+    return translations.get(normalized_place)
+
+
 def insert_rawis(conn: sqlite3.Connection, rawis_df: pl.DataFrame) -> None:
+    # Load place translations
+    with open(Path("data/place_translations.json")) as f:
+        place_translations = json.load(f)
+
+    # Create a function to translate places
+    translate_place_fn = lambda x: translate_place(x, place_translations)
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `translate_place_fn` as a `def`

(E731)